packages/utilities/crypto/lib/types.web.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Represents a public cryptographic key
+ */
+export interface PublicKey {
+	/** Returns a did:key representation of the public key */
+	did(): Promise<`did:key:${string}`>;
```

Comment:
I like the new `exportPublicKey` names, but why not move `did()` there too? (since that's also just a public key serialisation format)

Reply:
It's quite arbitrary tbf, it somehow felt a bit special and I figured having a method "just for it" wouldn't hurt. I'm unsure how relevant it is to consider getting the did repr a special case (more common than the others?), not wrangled with atp for long enough to have a good picture 🤔

Reply:
seems fine to me